src/core/Form/SearchInput/SearchInput.tsx

```diff
@@ -45,49 +45,54 @@ export interface SearchInputProps
       | 'value'
       | 'defaultValue'
     > {
-  /** SearchInput container div class name for custom styling. */
+  /** Custom classname to extend or customize */
```

Comment:
Should className texts "Custom classname to extend or customize" also be "CSS class for custom styles" like in other components?

Reply:
This is now addressed in the latest commit